src/assets/guide-markdown/data-requirements.md

```diff
@@ -25,45 +25,47 @@ or
 
 The Human Cell Atlas Data Coordination Platform accepts datasets featuring both data access types, with an important note on the data and metadata files required for each type:
 
-## Open access accepted data and metadata:
+## Open access datasets - accepted data and metadata:
 
-Raw sequencing files (fastq, bam)
+* Raw sequencing files (fastq, bam)
```

Comment:
it can be hard to distinguish between bold `**content**` italic `*content*` and bullet `* content`
So the more specific `- Content` is more widely used.
```suggestion
- Raw sequencing files (fastq, bam)
```